webpack/components/SelectOrg/SetOrganization.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Form, Button } from 'patternfly-react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import Select from 'foremanReact/components/common/forms/Select';
+import * as SelectOrgActions from './SelectOrgAction';
+import reducer from './SelectOrgReducer';
+import { LoadingState } from '../../move_to_pf/LoadingState';
+import './SelectOrg.scss';
+
+class SetOrganization extends Component {
+  constructor(props) {
+    super(props);
+    this.onSelect = this.onSelect.bind(this);
+    this.onSend = this.onSend.bind(this);
+    this.state = { disabled: true };
+  }
+
+  componentDidMount() {
+    this.props.getOrganiztionsList();
+  }
+
+  onSelect(e) {
+    this.setState({
+      selected: `${e.target.value}-${e.target.textContent}`,
+      disabled: false,
+    });
+  }
+
+  onSend() {
+    const {
+      history,
+      location,
+      changeCurrentOrgaziation,
+    } = this.props;
+
+    changeCurrentOrgaziation(this.state.selected)
+      .then(() => history.push(location.state));
+  }
+
+  render() {
+    const {
+      list,
+      loading,
+    } = this.props;
+
+    return (
+      <div className="well col-sm-6 col-sm-offset-3">
+        <LoadingState loading={loading} loadingText={__('Loading')}>
+          <Form>
+            <h1 className="center-header">{__('Select an Organization')}</h1>
```

Comment:
Using `.text-center` works here instead of `.center-header` and `.center-paragraph`, no idea why custom classes were created for this in angular katello.